pymobiledevice3/remote/core_device_tunnel_service.py

```diff
@@ -33,7 +35,10 @@
 from cryptography.hazmat.primitives.ciphers.aead import ChaCha20Poly1305
 from cryptography.hazmat.primitives.kdf.hkdf import HKDF
 from opack import dumps
-from pytun_pmd3 import TunTapDevice
+if sys.platform != 'win32':
+    from pytun_pmd3 import TunTapDevice
+else:
+    from pywintun_pmd3 import TunTapDevice
```

Comment:
this package should both be added to project's requirements and uploaded to pypi